### DISCUSSION TITLE
IOTA'larim yok oldu

### DISCUSSION THREAD

**Discussion by SSalpi** [iotaledger/firefly#8427] [Help & Support]
https://www.iotalottery.org/ sitenizde yedekleme dosyası ile firefly cüzdanımı bağladım ve sonra mevcut bana ait IOTA (Yaklaşık 2000 Dolar değerinde) bir anda kayboldu lütfen yardım edin.
Ayrıca Hediye olarak gelmiş olan 3 adet "You won 2000 iota" 'ları da alamadım.
Firefly Cüzdan Adresim iota1qr356hgjseywh0jt4vvcj23y8q0qv2jn6w4dx8hh3q6lzr6vx4c9xlvmeeg

**Comment by HankMoody089** [comment]
Thats a scam!!! Don't visit the page linked!

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
1) In your wallet go to: Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
4) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/5ecce50b-92af-4a09-97e8-5f72cf1bf9b8)
